datumaro/components/errors.py

```diff
@@ -197,21 +197,66 @@ class DatasetImportError(DatumaroError):
     pass
 
 
+class InvalidAnnotationError(DatasetImportError):
+    """
+    A basic dataset parsing error. Should include the problem description in
+    the message.
+    """
+
+
+@define(auto_exc=False)
+class InvalidFieldError(InvalidAnnotationError):
+    name: str
+
+    def __str__(self) -> str:
+        return f"Invalid anotation field {self.name} value"
```

Comment:
```suggestion
        return f"Invalid annotation field {self.name} value"
```

There's a few other instances of this mistake, as well.

Reply:
Fixed